eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -89,206 +87,202 @@ stages:
         container: ${{ parameters.alpine317Container }}
-        buildFromArchive: false            # ✅
-        enablePoison: false                # 🚫
-        excludeOmniSharpTests: true        # ✅
-        runOnline: false                   # 🚫
-        useMonoRuntime: false              # 🚫
-        withPreviousSDK: false             # 🚫
-
-    - template: ../jobs/vmr-build.yml
-      parameters:
-        # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-        buildName: CentOSStream8_Online_PreviousSourceBuiltSdk
-        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-        vmrBranch: ${{ variables.VmrBranch }}
-        architecture: x64
-        pool:
-          name: ${{ variables.defaultPoolName }}
-          demands: ${{ variables.defaultPoolDemands }}
-        container: ${{ parameters.centOSStream8Container }}
-        buildFromArchive: false            # 🚫
-        enablePoison: false                # 🚫
-        excludeOmniSharpTests: true        # ✅
-        runOnline: true                    # ✅
-        useMonoRuntime: false              # 🚫
-        withPreviousSDK: true              # ✅
-
-    - template: ../jobs/vmr-build.yml
-      parameters:
-        # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-        buildName: CentOSStream8_Offline_PreviousSourceBuiltSdk
-        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-        vmrBranch: ${{ variables.VmrBranch }}
-        architecture: x64
-        pool:
-          name: ${{ variables.defaultPoolName }}
-          demands: ${{ variables.defaultPoolDemands }}
-        container: ${{ parameters.centOSStream8Container }}
         buildFromArchive: false            # 🚫
-        enablePoison: false                # 🚫
+        enablePoison: true                 # ✅
         excludeOmniSharpTests: true        # ✅
         runOnline: false                   # 🚫
         useMonoRuntime: false              # 🚫
         withPreviousSDK: true              # ✅
 
-    - template: ../jobs/vmr-build.yml
-      parameters:
-        # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
-        buildName: CentOSStream8_Mono_Offline_MsftSdk
-        isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
-        vmrBranch: ${{ variables.VmrBranch }}
-        architecture: x64
-        pool:
-          name: ${{ variables.defaultPoolName }}
-          demands: ${{ variables.defaultPoolDemands }}
-        container: ${{ parameters.centOSStream8Container }}
-        buildFromArchive: true             # ✅
-        enablePoison: false                # 🚫
-        excludeOmniSharpTests: true        # ✅
-        runOnline: false                   # 🚫
-        useMonoRuntime: true               # ✅
-        withPreviousSDK: false             # 🚫
+    - ${{ if eq(variables['Build.DefinitionName'], 'dotnet-dotnet') }}:
```

Comment:
I'd prefer not having this tied to a specific name and instead pass this knowledge as a parameter to the template. So this template could define a parameter named `useFullBuild` that defaults to `false` and is set to true in ci.yml.

Reply:
Addressed in https://github.com/dotnet/installer/pull/17370/commits/e51a67e0c0aada67b320ce21b78a3ecd49ce8188:
- https://github.com/dotnet/installer/pull/17370/commits/e51a67e0c0aada67b320ce21b78a3ecd49ce8188#diff-681076b03a7ed5a0cd4fca97e3de0303bb65604b0d20e05885f35332dd1a18ebR12
- https://github.com/dotnet/installer/pull/17370/commits/e51a67e0c0aada67b320ce21b78a3ecd49ce8188#diff-681076b03a7ed5a0cd4fca97e3de0303bb65604b0d20e05885f35332dd1a18ebR133